Ginger/Ginger/ApplicationModelsLib/POMModels/PomAllElementsPage.xaml.cs

```diff
@@ -144,7 +144,7 @@ private void CreateNewElemetClicked(object sender, RoutedEventArgs e)
         }
 
 
-        System.Windows.Threading.DispatcherTimer dispatcherTimer = null;
+        public static System.Windows.Threading.DispatcherTimer mDispatcherTimer = null;
 
```

Comment:
1- why mDispatcherTimer is static? thoretically users can work on 2 POM pages in tree in parallel if each is connected to diffrent Agent. 
2- mDispatcherTimer  should not be Public and be touched from outside, instead expose "StopSpy()" public method which internally will make it False if needed